pkg/defaults/defaults.go

```diff
@@ -23,7 +25,7 @@ var (
 
 // Holds the default no proxy values.
 var (
-	DefaultNoProxy = []string{"localhost", "127.0.0.1", ".default", ".local", ".svc", "kubernetes", "kotsadm-rqlite", "10.0.0.0/8"}
+	DefaultNoProxy = []string{"localhost", "127.0.0.1", ".default", ".local", ".svc", "kubernetes", "kotsadm-rqlite", v1beta1.DefaultNetwork().ServiceCIDR, v1beta1.DefaultNetwork().PodCIDR}
```

Comment:
I'm approving despite this still being less than ideal - I'd prefer for this to include the CIDRs for the config being used in the cluster, as they might be overridden by the vendor/user in unsupported overrides

Reply:
Agree, I think this can be addressed in a follow up PR.

Reply:
Yes. We mentioned that once overriding CIDRs is supported, we should also support it.